Report/Fluvoxamine_evaluation_report.md

```diff
@@ -0,0 +1,403 @@
+# Building and evaluation of a PBPK model for Fluvoxamine in adults
+
+
+
+| Version                                         | 1.1-OSP10.0                                                  |
+| ----------------------------------------------- | ------------------------------------------------------------ |
+| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/releases/tag/v1.0 |
```

Comment:
1.0 is obviously wrong. 
Actually it should be 1.2 and become part of the new release 1.2 of this model (which is not created yet)

Reply:
Option 1: point to the branch: https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/tree/cyp1a2_cyp2C19_DDI 
Option 2: create a tag without release from this branch and point to it
Option 3: create a new Release (name it so that it's clear that this is "another Fluvoxamin") and point to it

I would at least create a tag, after all the issues are fixed

Reply:
Hen-egg-problem - how do I get the correct link for the tag before creating it? :D

Reply:
Lol. If you know how you are going to name your tag: 
[https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/tree/**TagName**]()

So if you name your tag **v1.1_FluvoxaminSpecial**, it would be
[https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/tree/**v1.1_FluvoxaminSpecial**](https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/tree/v1.1_FluvoxaminSpecial)